### DISCUSSION TITLE
Add better build instructions

### DISCUSSION THREAD

**Discussion by marcnoon** [dotnet/runtime#39017] [General] [answered]
Could you please add better build instructions.   The current build instructions don't appear to work or are unclear on how to build for each environment.  Could you also please add full build instructions for a Windows 10 environment for the dotnet runtime.

**Comment by stephentoub** [comment, marked as answer]
@marcnoon, can you please clarify what is unclear or what doesn't work?

**Reply by danmoseley** [reply]
https://github.com/dotnet/runtime/issues/39018?

**Reply by marcnoon** [reply]
Never mind... I didn't realize each of these github projects was relative to other projects by the dotnet/*NAME*.   So I think I need at a minimum ['build', 'corefx', 'runtime']  to build the projects I'm currently interested.     I also assumed each project could stand alone and realized the errors in that thinking.  -Cheers
